container/tests/java/javaCodeCoverageShould.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+computeCoverageForChallenge() {
+   # given
+   language_id=$1
+   tag=$2
+   repo="https://github.com/julianghionoiu/tdl-runner-${language_id}"
+   challenge_id="$3"
+   expectedResult="$4"
+   participant_id="participant"
+   round_id="round"
+   (cd ../.. && ./buildDockerImage.sh ${language_id})
```

Comment:
Do not build the containers. Only check if they are present in the Docker registry